common/src/main/java/org/jboss/hal/testsuite/fragment/shared/util/ResourceManager.java

```diff
@@ -93,4 +122,21 @@ public void viewByName(String name) {
         return getResourceTable().getTextInColumn(0);
     }
 
+    /**
+     * Checks if given resource is present in resource table
+     * @param name name of resource
+     * @return true if resource is present, false otherwise
+     */
+    public boolean isResourcePresent(String name) {
```

Comment:
Not sure regarding the timeouts, there might be worth testing that the item is there immediately, but OK.